hushline.nginx.j2

```diff
@@ -0,0 +1,46 @@
+# Redirect HTTP to HTTPS
+server {
+    listen 80;
+    server_name {{ domain }};
+    return 301 https://{{ domain }}$request_uri;
+}
+
+# HTTPS server block
+server {
+    listen 443 ssl http2; # Enable HTTP/2 if supported
+    server_name {{ domain }};
+
+    # SSL configuration
+    ssl_certificate /etc/letsencrypt/live/{{ domain }}/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/{{ domain }}/privkey.pem;
+    # Include additional SSL parameters from a common file (optional)
+    include snippets/ssl-params.conf;
+
+    # Root directory
+    root /var/www/html/hushline;
+
+    # Proxy pass configuration for handling requests
+    location / {
+        proxy_pass http://unix:/var/www/html/hushline/hushline.sock;
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
```

Comment:
If we're using CF for production, we don't want to `set` these headers since the real IP would be set to the CF IPs and thus would break all our rate limiting

Reply:
Resolving as we've moved, at least for now, away from CF.